internal/pkg/template/templates/addons/s3/env/cf.yml

```diff
@@ -0,0 +1,53 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The name of the environment being deployed.
+
+Resources:
+  {{logicalIDSafe .Name}}Bucket:
+    Metadata:
+      'aws:copilot:description': 'An Amazon S3 bucket, {{.Name}}, for storing and retrieving objects'
+    Type: AWS::S3::Bucket
+    Properties:
```

Comment:
Can we add this to our S3 bucket templates too 🥺 
https://github.com/aws/copilot-cli/blob/8d4f503dfb14a7e5985831aa191fc93175fbe6b7/internal/pkg/template/templates/app/cf.yml#L92-L94

It's okay if the service addon one is done separately

Reply:
Added this to both env addon and workload addon!